build_openwebtext_pretraining_dataset.py

```diff
@@ -79,11 +80,23 @@ def main():
                       help="Number of tokens per example.")
   parser.add_argument("--num-processes", default=1, type=int,
                       help="Parallelize across multiple processes.")
+
+  # toggle lower-case
   parser.add_argument("--do-lower-case", dest='do_lower_case',
                       action='store_true', help="Lower case input text.")
   parser.add_argument("--no-lower-case", dest='do_lower_case',
                       action='store_false', help="Don't lower case input text.")
+
+  # toggle strip-accents
+  parser.add_argument("--do-strip-accents", dest='strip_accents',
+                      action='store_true', help="Strip accents (default).")
+  parser.add_argument("--no-strip-accents", dest='strip_accents',
+                      action='store_false', help="Don't strip accents.")
+
+  # set defaults for toggles
   parser.set_defaults(do_lower_case=True)
+  parser.set_defaults(strip_accents=True)
```

Comment:
should we set the default to False here and other places to preserve the original behavior?